### ISSUE TITLE
"Bridges/Add bridge" appears to be missing

### ISSUE THREAD

**Issue by CryptoGuppy** [elgatosf/streamdeck-philipshue#5]
I installed the Stream Deck yesterday, and tried to add buttons to use with my Hue bridge/lights. However, when trying to select/add the bridge, I got the response "No bridge detected" (or something similar). I figured it had something to do with the fact that I run a VPN client. This VPN client allows for split tunneling (allowing certain apps to bypass the tunnel) so I added it to the exclude list. Today I wanted to try again, but it appears the bridge options have disappeared. When I select a Hue button, the only field shown is the "Title" field. I have already uninstalled the Hue plug-in and reinstalled it, but that didn't help. Uninstalling it, and re-installing it from the github file (instead of using the editor) didn't help either.
Streamdeck XL, firmware version 1.00.006, software version 4.5.1.12246,  Hue plug-in version 1.1.3
Hue App shows everything as up to date.
![Screenshot_380](https://user-images.githubusercontent.com/52297974/75094178-294cc900-5589-11ea-85c2-16cea50ea270.png)

**Comment by Timac** [comment]
Since you reported this issue, we released Stream Deck 4.7 which might have addressed this issue. Do you still encounter the problem?

**Comment by CryptoGuppy** [comment]
Yes, and no. The problem still occurs while my VPN is active, but if I deactivate it, the Hue hub (and the options) become visible. Once I have added and configured a Hue button on my SD, it works regardless of the VPN. The problem only occurs when I try to add/configure a button. I noticed this problem with a lot of the Stream Deck plugins (I am unable to download/install a new plugin when VPN is active), I suspect the IP addresses the VPN server uses are on a blacklist (when browsing, I quite often have to complete a captcha because of "suspicious traffic").
Anyway, for me personally, the problem is solved (I can add the buttons), and this is probably just one of those fringe cases. No need to keep this issue open for me.
